assign3/script.js

```diff
@@ -0,0 +1,42 @@
+document.getElementById("submit").addEventListener("click", submitNewTask);
+
+function submitNewTask() {
+  const inputTask = {
+    id: parseInt(document.querySelector(".task_list").rows.length - 1),
+    comment: document.getElementById("comment").value,
+    status: "作業中",
+    delete: "削除",
+  }
+  createNewTaskElement(inputTask);
+}
+
+function createTextCell(parent, value) {
+  const childNode = document.createElement("td");
+  const cell = document.createTextNode(`${value}`);
+  childNode.appendChild(cell);
+  parent.appendChild(childNode);
+}
+
+function createButtonCell(parent, v1, v2) {
```

Comment:
引数の名前がやや抽象的と思うので、より具体的な名前をつけて欲しいです。
```suggestion
function createButtonCell(parent, id, buttonValue) {
```

Reply:
修正しました。